src/main/java/ladder/domain/LadderGame.java

```diff
@@ -0,0 +1,26 @@
+package ladder.domain;
+
+import ladder.domain.dto.LadderShapeResult;
+import ladder.domain.dto.StairDto;
+import ladder.domain.ladder.Height;
+import ladder.domain.ladder.Ladder;
+import ladder.domain.player.Players;
+
+public class LadderGame {
+
+    private final Players players;
+    private final Ladder ladder;
+
+    private LadderGame(final Players players, final Height height) {
+        this.players = players;
+        this.ladder = Ladder.of(height, players.count());
+    }
+
+    public static LadderGame of(final Players players, final Height height) {
+        return new LadderGame(players, height);
+    }
+
+    public LadderShapeResult play() {
+        return LadderShapeResult.of(players.getNames(), StairDto.from(ladder));
```

Comment:
```suggestion
        return LadderShapeResult.of(players.getNames(), ladder);
```
이렇게 하고 LadderShapeResult 객체 내부에서 StairDto를 생성하는것도 나쁘지 않을거 같아요.
LadderGame에서 의존하는 객체가 줄어들게 되니까요